e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/change-your-answers-change-broker.spec.js

```diff
@@ -196,7 +196,8 @@ context('Insurance - Change your answers - Policy - Broker - Summary list', () =
 
       it('should render the new answer and retain a `completed` status tag', () => {
         fieldVariables.newValue = fieldVariables.newValueInput;
-        cy.checkChangeAnswerRendered(fieldVariables);
+        // 1 as is the second email field on the page
+        cy.checkChangeAnswerRendered(fieldVariables, 1);
```

Comment:
Are the 2 email fields - 1 for broker and 1 for policy contact using the same `EMAIL` ID?

I'm wondering if it'd be better to select the the fields via the child summary list (for X section), rather than "second email field in the whole page".

Otherwise not sure if should be tech debt, but `checkChangeAnswerRendered` would be nicer in an object structure so `1` has more context and then no need for a comment

Reply:
The functions not able to do this yet I think and it would be a whole other PR to get it working in that way.  I think it's better to do it in another PR as Caroline is blocked on testing by these PRs

Reply:
It seems that all `checkChangeAnswerRendered` commands (~20) could be updated very easily to support this with object structure. 5min job?

Any existing commands e.g `cy.checkChangeAnswerRendered(fieldVariables);` would still work with new structure. just in this new pattern it would be nicer and not introducing tech debt or additional code comments 

e.g
```js
cy.checkChangeAnswerRendered({ ...fieldVariables, index: 1 });
```